developing_packages/editable_packages.rst

```diff
@@ -24,32 +24,40 @@ to copy locally built artifacts into the conan cache and make them available to
 With the editable packages, you can tell Conan where to find the headers and the artifacts ready for
 consumption in your local working directory. There is no need to package.
 
-Let's see this feature over an example where a developer is creating a ``CoolApp`` but at the same
-time they want to work on ``cool/version@user/dev`` library which is tightly coupled to the app.
+Let's see this feature over an practical example; the code can be found in the conan examples repository:
 
-The package ``cool/version@user/dev`` is already working, the developer has the sources in a
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-io/examples.git
+    $ cd features/editable/cmake
```

Comment:
Yeah, but I followed the same idea from other examples in docs. I don't think putting `cd` in the description is problem, because it works for any OS.

Reply:
No, but if we assume that the user is already inside the directory `features/editable/cmake`, then all the other commands in this piece of docs wouldn't be so clear (IMO): instead of

```
conan editable add examples/features/editable/cmake/say say/0.1@user/channel
```

it should be

```
conan editable add say say/0.1@user/channel
```

That's my only point to avoid that `cd`

Reply:
I've removed cd from the docs